keywords/couchbaseserver.py

```diff
@@ -58,6 +58,30 @@ def verify_server_version(host, expected_server_version):
         raise ProvisioningError("Unsupported version format")
 
 
+def create_internal_rbac_bucket_user(url, bucketname):
+    # Create user with username=bucketname and assign role
+    # bucket_admin and cluster_admin
+    roles = "cluster_admin,bucket_admin[{}]".format(bucketname)
```

Comment:
I added a function to delete users and it gets called when bucket gets deleted. We can check if the user exists but not if the password is what we are expecting